src/CTA.WebForms2Blazor/FileConverters/ViewFileConverter.cs

```diff
@@ -1,26 +1,93 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.IO;
 using System.Linq;
+using System.Text;
 using System.Threading.Tasks;
+using CTA.WebForms2Blazor.ControlConverters;
 using CTA.WebForms2Blazor.FileInformationModel;
+using CTA.WebForms2Blazor.Helpers;
+using HtmlAgilityPack;
 
 namespace CTA.WebForms2Blazor.FileConverters
 {
     public class ViewFileConverter : FileConverter
     {
-        public ViewFileConverter(string sourceProjectPath, string fullPath) : base(sourceProjectPath, fullPath)
+        private Dictionary<String, ControlConverter> ControlRulesMap; //From constructor or service
+        //Tuple contains node to be converted, parent node, and instructions to convert node in that order
+        private List<(HtmlNode Node, HtmlNode Parent, ControlConverter Rules)> NodesToBeConverted;
+        private string _relativeDirectory;
+
+        public ViewFileConverter(string sourceProjectPath, string fullPath, Dictionary<String, ControlConverter> dict) 
+            : base(sourceProjectPath, fullPath)
         {
             // TODO: Register file with necessary services
+            _relativeDirectory = Path.GetDirectoryName(RelativePath);
+            NodesToBeConverted = new List<(HtmlNode Node, HtmlNode Parent, ControlConverter Rules)>();
+            ControlRulesMap = dict;
+        }
+
+        private HtmlDocument GetRazorContents()
+        {
+            var htmlDoc = new HtmlDocument();
+            htmlDoc.Load(FullPath);
+            DfsPostorderWalker(htmlDoc.DocumentNode, null);
+            
+            //This will either return string contents or modified HtmlDocument
+            //that will then be changed to a file information object
+            
+            ConvertNodes();
+            return htmlDoc;
+        }
+
+        private void DfsPostorderWalker(HtmlNode node, HtmlNode parent)
+        {
+            if (node == null)
+            {
+                return;
+            }
+
+            foreach (HtmlNode child in node.ChildNodes)
+            {
+                DfsPostorderWalker(child, node);
+            }
+
+            GetRules(node, parent);
+        }
```

Comment:
Would improve readability to rename closer to what the method is doing, e.g. `FindConversionActions` or something similar. It would help to add a doc comment describing how the rules are being found (i.e. DFS through tree to identify nodes in postorder fashion).